### DISCUSSION TITLE
Second Edition

### DISCUSSION THREAD

**Discussion by Felderburg** [bcholmes/StarTrek2d20#210] [General]
Just figured I'd start a discussion for/about the upcoming 2nd edition of STA.

**Comment by aecolesuk** [comment]
We got the PDF yesterday. My live stream group will be running STA 2e as soon as I get the physical book.
Hoping the character creator gets updated relatively shortly. Is there a way to be a beta tester for it?

**Reply by matthew81** [reply]
Same, So many small changes gonna be confusing for a while.

**Reply by LatiusAuro** [reply]
Yeah, at least compared to the original base edition - it gives more species in the starting list, but it also gives each species a specefic ability in place of a starting trait, just to start.

**Reply by Felderburg** [reply]
They get the ability *in addition* to a trait. See page 98, the Lifepath Summary, step one.
